system-addon/test/functional/mochitest/browser_highlights_section.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+async function add_highlights_bookmark(count) {
+  const bookmarks = new Array(count).fill(null).map((entry, i) => ({
+    parentGuid: PlacesUtils.bookmarks.unfiledGuid,
+    title: "foo",
+    url: `https://mozilla${i}.com/nowNew`
+  }));
+
+  for (let placeInfo of bookmarks) {
+    await PlacesUtils.bookmarks.insert(placeInfo);
+  }
+
+  // Add bookmark visits
+  for (let placeInfo of bookmarks) {
+    await PlacesTestUtils.addVisits(placeInfo.url);
+  }
+}
+
+// Test populating highlights section with bookmarks.
+add_task(async function getHighlights() {
+  const count = 2;
+
+  await setUpActivityStreamTest();
+  await add_highlights_bookmark(count);
+
+  let selector = ".card-outer:not(.placeholder)";
```

Comment:
I'm not the hugest fan of using specific style-related classes for tests because it makes refactoring kind of a pain. Do we want to maybe go with an attribute-based approach or something like that, or maybe at least use a constant to store this selector with a comment in the source code? Any suggestions @Mardak?